plugins/services/src/js/containers/services/ServicesContainer.js

```diff
@@ -688,6 +701,10 @@ class ServicesContainer extends React.Component {
       return this.getServiceTree(item);
     }
 
+    if (currentRoutePath.startsWith("/services/quota")) {
```

Comment:
the render-method could definitely benefit from a tell-don't-ask-refactoring. 🙈

Reply:
I agree, but if you look above to line 694 this pattern was already used. I was trying to match the existing code.

Reply:
i'm not asking to change that here and now as the PR is pretty old/big already. 

as we're good boyscouts and clean stuff up no matter who made the mess in the first place i still wanted to point that out. :)